src/ert/run_models/run_arguments.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class SingleTestRunArguments:
+    simulation_mode: str = field(init=False, default="Single test run")
+
+    active_realizations: List[bool]
```

Comment:
I hate the prefix "active". Everything is "active" is not informative. Maybe just `realizations` or `realizations_mask`?

Reply:
I agree that `active` is not great, but I don't think we should make this change in this PR.